src/qml/pages/NewChatPage.qml

```diff
@@ -9,64 +10,288 @@ import "../theme"
+                                            anchors.fill: parent
+                                            radius: parent.height
+                                        }
+                                    }
+                                }
+
+                                Image {
+                                    anchors.fill: parent
+                                    source: model.avatarUrl
+                                    mipmap: true
+                                    asynchronous: true
+                                    fillMode: Image.PreserveAspectCrop
+                                }
+                            }
+                        }
+                    }
+
+                    Column {
+                        Layout.fillWidth: true
+                        clip: true
+
+                        Text {
+                            color: Theme.primaryTextColor
+                            font.pointSize: UiHelper.fixFontSz(15)
+                            text: model.name
+                        }
+
+                        Text {
+                            color: Theme.secondaryTextColor
+                            font.pointSize: UiHelper.fixFontSz(12)
+                            text: model.lastSeenActivity
+                            width: parent.width
+                            elide: Text.ElideRight
+                            textFormat: Text.RichText
+                        }
+                    }
+
+                }
+
+            }
+
         }
+    }
 
-        RowLayout {
-            Layout.fillWidth: true
-            Layout.maximumWidth: Theme.formMaximumWidth
-            Layout.alignment: Qt.AlignHCenter
+    Item {
+        anchors {
+            top: searchResultsItem.bottom
+            left: parent.left
+            right: parent.right
+            bottom: parent.bottom
+            leftMargin: defaultMargins
+            rightMargin: defaultMargins
+        }
+
+        Form {
+            id: form
+            formSpacing: 25
+            focus: true
 
-            Label {
-                text: qsTr("Server")
-                color: Theme.primaryTextColor
+            RowLayout {
                 Layout.fillWidth: true
-            }
+                Layout.maximumWidth: Theme.formMaximumWidth
+                Layout.alignment: Qt.AlignHCenter
+
+                Label {
+                    text: qsTr("Server")
+                    color: Theme.primaryTextColor
+                    Layout.fillWidth: true
+                }
 
-            Label {
-                color: Theme.secondaryTextColor
-                text: qsTr("Default")
+                Label {
+                    color: Theme.secondaryTextColor
+                    text: serverName
+                }
             }
         }
+    }
 
-        FormPrimaryButton {
-            text: qsTr("Add to contacts")
-            onClicked: accept()
+    function checkState() {
```

Comment:
This doesn't work as expected. If I type "dav" it displays "David" only

Reply:
I see. Changing logic

Reply:
Fixed